src/software/embedded/robot_localizer_test.cpp

```diff
@@ -0,0 +1,57 @@
+#include "software/embedded/robot_localizer.h"
+
+#include <gtest/gtest.h>
+
+TEST(BasicTest, basic_test2) {
```

Comment:
```suggestion
TEST(RobotLocalizerTest, ...) {
```
the test name should be clearer